common/web/input-processor/tests/cases/inputProcessor.js

```diff
@@ -79,7 +79,7 @@ describe('InputProcessor', function() {
 
       // These two lines will load a keyboard from its file; headless-mode `registerKeyboard` will
       // automatically set the keyboard as active.
-      let kbdScript = new vm.Script(fs.readFileSync('../tests/resources/keyboards/test_chirality.js'));
+      let kbdScript = new vm.Script(fs.readFileSync('../../test/resources/keyboards/test_chirality.js'));
```

Comment:
Doesn't look like the new path is correct if it refers to a file in `common/test/resources`. I think it's missing one additional `../`.

Reply:
I'm pretty sure the paths are correct as they are -- the tests pass so the keyboards are definitely being loaded.

Note that the path is relative to the build folder and not the source folder for the script...